core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/named/AddNamedViewHandler.java

```diff
@@ -37,19 +37,19 @@
 /**
  * Operation handler for {@link AddNamedView} which adds a NamedViewDetail to the cache.
  */
-public class AddNamedViewHandler implements OperationHandler<AddNamedView> {
+public class AddNamedViewHandler implements OperationHandler<AddNamedView>, AddToCacheHandler<AddNamedView> {
```

Comment:
See other comments about Y being redundant when implementing X and Y, where Y is parent of interface X.

Reply:
yeah was just being verbose to be clear.
Shall I remove?